### DISCUSSION TITLE
How to get the YOLO badge?

### DISCUSSION THREAD

**Discussion by thesmilingsabre** [community/community#18611] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I get the new YOLO GitHub achievement?

**Comment by brunomdrrosa** [comment, marked as answer]
You just have to merge a pull request without code review.

https://github.com/Schweinepriester/github-profile-achievements

**Reply by rahmatsubandi** [reply]
Hi, what is no code review? I've tried a lot but still can't? Can you provide a reference for me?

**Reply by Thedragcoder** [reply]
huh:

**Reply by astianmuchui** [reply]
Work on a project with another user , then merge their pull request without reveiws

**Reply by ghost** [reply]
> Work on a project with another user , then merge their pull request without reveiws

https://github.com/orgs/github-community/discussions/18611#discussioncomment-3119759

**Reply by anandguptacv** [reply]
yolo is for?

**Reply by sundios** [reply]
You only live once = YOLO

**Reply by KissesJun** [reply]
> Yolo is for?
@sundios said:
You only live once😅

**Reply by Khairul989** [reply]
thanks

**Reply by malyshevio** [reply]
:heart:  thank you

**Reply by truongtran456** [reply]
ok

**Reply by Trogluddite** [reply]
yep that worked! Now let's see if making a second community comment gets me the galaxy badge ... 🤔

**Reply by VitorVargass** [reply]
Thanks man

**Reply by ghost-viper2607** [reply]
thanks.

**Reply by AdRohal** [reply]
Thank youu❤️

**Reply by ShekhSujan** [reply]
Thanks  ❤️

**Comment by anupbasnet19** [comment]
Follow the below steps.
1. Create a pull request on your own remote repository.
2. Open the pull request.
3. At the top right corner add a reviewer to your repository.
4. Before they review your code and merge you should merge your code to the remote repository.

**Reply by williamvietnam** [reply]
if i reply in here, can me receiver bage ?

**Reply by geraldokereke** [reply]
Which badge?

**Reply by masbagoel** [reply]
Thanks, its works

**Reply by Bitcoinera** [reply]
It works! Why is this not the accepted answer? Finally!!

**Reply by rocconeto** [reply]
Yes, It works!

**Reply by nguyenary** [reply]
It works

**Reply by SamX23** [reply]
Not works, or it is a cache on me

**Reply by AdityaArgadinata** [reply]
got it

**Reply by Phant0m-a** [reply]
> Follow the below steps.
> 
> 1. Create a pull request on your own remote repository.
> 2. Open the pull request.
> 3. At the top right corner add a reviewer to your repository.
> 4. Before they review your code and merge you should merge your code to the remote repository.
> 5. get job

**Reply by Phant0m-a** [reply]
for some reason its not working

**Reply by arijhajji-1** [reply]
thank you

**Reply by taotechs** [reply]
It worked

**Reply by JustGimli** [reply]
thanks

**Reply by JustGimli** [reply]
it is work

**Reply by zcrossoverz** [reply]
It works! Thank you

**Reply by truongtran456** [reply]
ok

**Reply by rover07** [reply]
That works fine!

**Reply by rover07** [reply]
Gettin my galaxy batch with this one.

**Reply by Trogluddite** [reply]
I would also like to get my galaxy badge, but will need to make a second comment after this one 👍

**Reply by joinemm** [reply]
awesome that works

**Reply by Reiaslite** [reply]
Thank you!

**Comment by themiralay** [comment]
for Galaxy badge: Answer two discussions
for Pull Shark badge : Have two pull requests merged
for yolo badge : Merge code without review
for more info : https://github.com/Schweinepriester/github-profile-achievements

**Reply by masbagoel** [reply]
Thanks, its works

**Reply by salecler** [reply]
Nice :)

**Reply by AdityaArgadinata** [reply]
thanks bro

**Reply by arijhajji-1** [reply]
thanks

**Reply by taotechs** [reply]
For the Galaxy?

**Reply by bdanzi** [reply]
Thank you for the hints!

**Reply by Jayantkhandebharad** [reply]
thanks

**Reply by highmountaindev109** [reply]
awesome

**Reply by NielsM05** [reply]
thanks

**Reply by bopzen** [reply]
thanks

**Reply by joinemm** [reply]
thanks

**Reply by Reiaslite** [reply]
Thank you!

**Reply by educlopez** [reply]
thanks!

**Reply by Visual-Studio-Coder** [reply]
thanks

**Comment by akhil-ge0rge** [comment]
You have to merge a pull request without code review

**Reply by malaidev** [reply]
You just have to merge a pull request without code review.

https://github.com/Schweinepriester/github-profile-achievements

**Reply by bopzen** [reply]
thanks

**Comment by thisShouldNotBeWorking** [comment]
1)Create a new branch in your repo
2)Make some changes in your code
3)Go to the pull request tab where you will see compare and pull request option
4)Open the pull request 
5)From the right side menu bar add a reviewer 
6)If you don't see any reviewers add a collaborator to your repo
(To add a collaborator go to settings ->access->collaborators and invite someone)
7)Now make the pull request
8)Finally Merge the pull request

**Reply by AdityaArgadinata** [reply]
it works thx

**Reply by WMRamadan** [reply]
Will this not work if you add yourself as the reviewer?

**Reply by naseerx** [reply]
Really helpful sir
Very well explain

**Reply by Mule-ME** [reply]
Thanks it works

**Reply by educlopez** [reply]
thanks

**Comment by muhibarshad** [comment]
you can pull request and merge it without the code review get the YOLO badge

**Comment by rocconeto** [comment]
> How do I get the new YOLO GitHub achievement?

You just have to merge a pull request without code review

**Comment by GuillermoFarias** [comment]
but, shouldn't this be a bad practice?
do a PR on a group project if you approve your changes yourself? doesn't make sense to me

**Reply by nicolasboisseau** [reply]
I agree. Having forever a badge for a bad practice you did once it's quite a heavy sanction.. lol

**Reply by thaian161** [reply]
i agree LOL

**Reply by ghost** [reply]
You’re absolutely right.

**Comment by aarabii** [comment]
> How do I get the new YOLO GitHub achievement?
You need to merge a pr without reviewing it. You can go do it in few seconds too
1.  Open any of your public PR 
2. open a file like `README.md` copy its content and delete that file
3. create a new branch and create a `README.md` file and merge it to your `master` or `main` branch 
4. thats it you have you acchivement

**Comment by bellodavid** [comment]
- Open any of your public PR
-open a file like README.md copy its content and delete that file
-create a new branch and create a README.md file and merge it to your master or main branch
thats it you have you achievement

**Comment by Itsfoss0** [comment]
You need to merge a PR without review.

1. Clone any of your remote repo
2. Create a new branch in your repo
3. Make changes to any file in the branch
4. Push the new branch to remote 
5. Compare and merge.

**Comment by malaidev** [comment]
You just have to merge a pull request without code review.
Here below;
https://github.com/Schweinepriester/github-profile-achievements

**Reply by whitetigerhunter** [reply]
Good code!

**Reply by john082Coder** [reply]
Hard coding.

**Comment by Kushal997-das** [comment]
Hello I got the answer and it's perfectly working -

STEP 1 : Choose a normal repo. of yours remember in that repo there shouldn't be any .github folder or any auto running things not present .

STEP 2 : Now go to that repository settings add another account as collaborator . (Settings->collaborator->Add people).

STEP 3 : You have to Create a Pull request (PR) on this repository.

STEP 4 : Add that collaborator account as reviewer.

STEP 5 : Then You have to merge your opned PR from Your account only not collaborator , Without reviewing the PR .

I hope it gonna help you guys , Thanks :)

**Comment by dhengGhuring** [comment]
You just merge your repo without a code review

**Reply by Behnamfe62** [reply]
> #~~function[]()vertical
- Function 5#[]()__~~

**Comment by ermiyasabate** [comment]
You just have to merge a pull request without code review.

https://www.youtube.com/watch?v=wBW-vjM7ccA&ab_channel=AnkushChauhan

**Comment by Abhishek-Suryavanshi** [comment]
You simply have to merge a pull request without code review

**Comment by sahana-github** [comment]
How to add reviewer in my pr

**Reply by itsmedeepu** [reply]
add reviewer when u make pull request and merge it without reviewer changes ✅✅

**Reply by sahana-github** [reply]
@itsmedeepu how to add reviewer?🧐

**Reply by itsmedeepu** [reply]
select repo in which u have to add reviewer then select settings-> then collabrators-->add username of collabrator ->if that added collabrator accepts the request .then that user will be available when  u make pull request in reviewers tab
✅✅

**Comment by SeyfCHERNI** [comment]
1)Create a new branch in your repo
2)Make some changes in your code
3)Go to the pull request tab where you will see compare and pull request option
4)Open the pull request
5)From the right side menu bar add a reviewer
6)If you don't see any reviewers add a collaborator to your repo
(To add a collaborator go to settings ->access->collaborators and invite someone)
7)Now make the pull request
8)Finally Merge the pull request

**Comment by tamnguyenmau** [comment]
You need to merge a PR without review.

1.Clone any of your remote repo
2.Create a new branch in your repo
3.Make changes to any file in the branch
4.Push the new branch to remote
5.Compare and merge.

**Comment by Rondev9** [comment]
You need to merge a pull request without review.
you can find much more information about the badges here: [https://github.com/Schweinepriester/github-profile-achievements](url)

**Reply by darkmoon-r** [reply]
Please explain more about it.
I did pull requests without review before, but couldn't get this badge.
Please help me with this.

**Comment by codemajesty999** [comment]
you are welcom

**Comment by ghost-viper2607** [comment]
repons #112592

**Comment by ettaboyle** [comment]
### Update for anyone following this thread

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by robinhood074** [comment]
@thesmilingsabre  You just have to merge a pull request without code review.
